### PR TITLE
README | typos

### PR DESCRIPTION
**Pull request by Vulwsztyn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -582,7 +582,7 @@ var outputPromise = getInputPromise().then(function (input) {
 
 * **Using Async Await**  
 
-Async await makes asynchronous code look like it\’s synchronous. This has only been possible because of the reintroduction of promises into node.js. Async-Await only works with functions that return a promise.
```
Comment (learning-zone) [inline README.md:585]: Escaping apostrophe is required here, becuase it prevent page break while deploying in GitHub Pages.

**Review by learning-zone — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Vulwsztyn) [reply README.md:585]: Should I add one to `it's`s in lines 1517 and 1551 then?

Comment (learning-zone) [reply README.md:585]: yep, it should be `it\'s`

Comment (Vulwsztyn) [reply README.md]: Changed

**Review by learning-zone — APPROVED** [review]
Approved

**Pull request merged**